src/ValueObject/Currency.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\ValueObject;
```

Comment:
Are we sure we want it in App\ValueObject directly, no specific context? @kuraobi what's your take?

Reply:
Any namespace that only describes the type of classes in it instead of the context/concept they belong to is not a good namespace. `ValueObject` is especially bad because it just says "these objects have no unique identifier", which really isn't very helpful because it could still be anything. I would suggest either a namespace for handling anything currency related, like `App\Currency`, or something wider like `App\G11n` or `App\L10n`, or a combination of both (`App\G11n\Currency`)...